src/test/java/email/KafkaRestClientUnitTest.java

```diff
@@ -0,0 +1,45 @@
+package email;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.companieshouse.api.strikeoffobjections.email.KafkaRestClient;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class KafkaRestClientUnitTest {
+
+    private String schemaRegistryUrl = "http://testSchema:1000";
+    private String emailSchemaUri = "/subjects/test-email-send";
+    private ResponseEntity response;
+    private String schemaUrl;
+
+    private KafkaRestClient restClient;
+    private RestTemplate restTemplate;
+
+    @BeforeEach
+    public void setUp() {
+        restTemplate = Mockito.mock(RestTemplate.class);
+        restClient = new KafkaRestClient(restTemplate);
+        schemaUrl = String.format("%s%s", schemaRegistryUrl, emailSchemaUri);
+        response = new ResponseEntity<>(HttpStatus.OK);
```

Comment:
Think Mark mentioned it, but as we are asserting on the response.body we should probably give it some content

Reply:
yes, in the constructor args, you should be able to add the body as the first param

Reply:
Added your response body